ui/tests/pages/components/http-requests-container.js

```diff
@@ -0,0 +1,5 @@
+import { selectable } from 'ember-cli-page-object';
+
+export default {
+  select: selectable('[data-test-timewindow-select]'),
```

Comment:
If this is the only thing the page-object is used for, you could swap to `fillIn` from ember-test-helpers and delete some things. https://github.com/emberjs/ember-test-helpers/blob/master/API.md#fillin is the api doc, it'd be `import { fillIn } from @ember/test-helpers` . 

The benefit of doing a page object would be that you can compose them and reuse them - these components are pretty specialized though so I'm not sure there's a lot of value in using page objects here.